stylesheet.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<xsl:stylesheet
+  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  version="1.0"
+>
+  <!-- Indent output -->
+  <xsl:strip-space elements="*" />
```

Comment:
This will remove a lot of blank lines. I like the consistent output (and that it encourages using comments for separation), but this might be an unpopular tradeoff.

Reply:
Yeah I suppose few formatting rules are universally popular - the comment argument seems like a reasonable response to any raised eyebrows